address.go

```diff
@@ -161,3 +165,28 @@ func (c *Client) GetAddressWithContext(ctx context.Context, addressID string) (o
 	err = c.get(ctx, "addresses/"+addressID, &out)
 	return
 }
+
+// CreateAndVerifyAddress Create Address object and immediately verify it.
+func (c *Client) CreateAndVerifyAddress(in *Address, opts *CreateAddressOptions) (out *Address, err error) {
+	req := &createAddressRequest{CreateAddressOptions: opts, Address: in}
+
+	response := AddressVerifyResponse{}
+
+	err = c.post(context.Background(), "addresses/create_and_verify", req, &response)
+
+	out = response.Address
+	return
+}
+
+// CreateAndVerifyAddressWithContext performs the same operation as CreateAndVerifyAddress, but allows
+// specifying a context that can interrupt the request.
+func (c *Client) CreateAndVerifyAddressWithContext(ctx context.Context, in *Address, opts *CreateAddressOptions) (out *Address, err error) {
+	req := &createAddressRequest{CreateAddressOptions: opts, Address: in}
+
```

Comment:
Let's not do line breaks between each of these lines just so we match the style of the rest of this repo.